src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -154,8 +188,17 @@ const WorkspaceDashboard = _.flow(
 
   const loadStorageCost = withErrorReporting('Error loading storage cost data', async () => {
     if (Utils.canWrite(accessLevel)) {
-      const { estimate } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
+      const { estimate, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
       setStorageCostEstimate(estimate)
+      setStorageCostEstimateUpdated(lastUpdated)
+    }
+  })
+
+  const loadBucketSize = withErrorReporting('Error loading bucket size.', async () => {
+    if (Utils.canWrite(accessLevel)) {
+      const { usageInBytes, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).bucketUsage()
+      setBucketSize(Utils.formatBytes(usageInBytes))
```

Comment:
Somewhat of a nit, but I think it would make sense to store these 2 things together since they are always fetched together.